packages/teams-js/src/public/meeting.ts

```diff
@@ -523,12 +548,21 @@ export namespace meeting {
   }
 
   /**
-   * Allows an app to share contents in the meeting
+   * Allows an app to share a given URL to the meeting stage for all users in the meeting.
+   * To learn more, visit https://aka.ms/teamsjs/shareAppContentToStage
    *
-   * @param callback - Callback contains 2 parameters, error and result.
-   * error can either contain an error of type SdkError, incase of an error, or null when share is successful
-   * result can either contain a true value, incase of a successful share or null when the share fails
-   * @param appContentUrl - is the input URL which needs to be shared on to the stage
+   * @remarks
+   * This API can only be used in the `sidePanel` and `meetingStage` frame contexts.
+   * For private scheduled meetings, meet now, or calls, include the `MeetingStage.Write.Chat` RSC permission in your app manifest.
```

Comment:
Could you link to appropriate manifest documentation? This is a great detail to include in here.

Reply:
I'm going to stop writing this every time I see it, but in general adding pointers to the manifest doc anytime you talk about the manifest in here would be great.

Reply:
Done!